platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/turtle/runner/Turtle.java

```diff
@@ -222,4 +225,16 @@ private void tickAllNodes() {
             }
         }
     }
+
+    public void validate() {
+        for (final TurtleNode node : nodes) {
```

Comment:
I think we can add additional checks like:

1. validity of creationTimestamp in SignedState (e.g.) to be > the start time of the test and < current time 
2. The SigSet to include only signatures from the current roster used in the round - this will be especially useful for the future DAB integration

If needed the SignedStateHolder can be enhanced with additional fields.